src/element-mixin.js

```diff
@@ -371,6 +350,14 @@ let nativeImportNode = Document.prototype.importNode;
 let nativeSetAttribute = Element.prototype.setAttribute;
 let nativeRemoveAttribute = Element.prototype.removeAttribute;
 
+export let setAttribute = function(attr, value) {
```

Comment:
Why not take the node as the first argument?
`setAttribute(node, attr, value)` would remove the indirection used later.